moped-database/metadata/tables.yaml

```diff
@@ -3150,6 +3171,52 @@
           - project_role_id
         filter: {}
         check: null
+  event_triggers:
+    - name: activity_log_moped_proj_personnel_roles
+      definition:
+        delete:
+          columns: '*'
+        enable_manual: false
+        insert:
+          columns: '*'
+        update:
+          columns: '*'
+      retry_conf:
+        interval_sec: 10
+        num_retries: 0
+        timeout_sec: 60
+      webhook_from_env: HASURA_ENDPOINT
+      headers:
+        - name: x-hasura-admin-secret
+          value_from_env: ACTIVITY_LOG_API_SECRET
+      request_transform:
+        body:
+          action: transform
+          template: |-
+            {
+                            "query": "mutation InsertActivity($object: moped_activity_log_insert_input!) { insert_moped_activity_log_one(object: $object) { activity_id } }",
+                            "variables": {
+                                "object": {
+                                    "record_id": {{ $body.event.data.new.project_personnel_id }},
+                                    "record_type":  {{ $body.table.name }},
+                                    "activity_id": {{ $body.id }},
+                                    "record_data": { "event": {{ $body.event }}},
+                                    "record_project_id": null,
```

Comment:
`moped_proj_personnel_roles` does not have a column for the project_id, so this is null. This also means that while we are tracking these activities, they will not show up in our activity log query since we query on project_id

Reply:
this is great—thanks for getting the tracking in place.

i also want to point out that, because we don't have the project ID here, we cannot update the project's modified date through this mutation. that's frustrating because i think user's should be able to expect that any project edit will update the project's modified date. if we wanted to get clever, we _could_ mutate the `moped_proj_personnel` record from here, which would then fire off another event which would mutate the parent project's modified date. that would require us adding some mutable field on `moped_proj_personnel` like an `updated_at` timestamp. 

just throwing it out there—i don't necessarily think this is a good idea. i believe we're also going to encounter this problem when we set up tracking for project features.